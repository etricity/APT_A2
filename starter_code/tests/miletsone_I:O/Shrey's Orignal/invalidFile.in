2
loadTest
4