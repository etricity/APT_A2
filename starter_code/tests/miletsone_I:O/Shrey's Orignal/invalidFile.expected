Welcome to Qwirkle!
-------------------


Menu
----
1. New Game
2. Load Game
3. Show student infomation
4. Quit

>2
Please enter a filename
>loadTest
Sorry the file does not exist

Menu
----
1. New Game
2. Load Game
3. Show student infomation
4. Quit

> 4
Goodbye!